app/__init__.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+
+from app.config import configs
+from app.utils.store import DocumentStore
+
+document_store = DocumentStore()
+
+from .download.views import download_blueprint
+from .upload.views import upload_blueprint
+
+
+def create_app(environment):
```

Comment:
Yeah, definitely. There's a lot of instrumentation that's missing right now. I want to get the basic working version in first, so that I can add the deployment pipeline to Jenkins and start adding the missing bits incrementally.